packages/jest-each/src/bind.ts

```diff
@@ -15,8 +14,8 @@ import {ErrorWithStack} from 'jest-util';
 
 type Table = Array<Array<any>>;
 type PrettyArgs = {
-  args: Array<mixed>,
-  title: string,
+  args: Array<any>;
```

Comment:
```suggestion
  args: Array<unknown>;
```

if it doesn't cause huge issues 🙂